.config.yml

```diff
@@ -2,6 +2,6 @@
 componentsGroupId: "org.mozilla.components"
 
 # Synchronized build configuration for all modules
-compileSdkVersion: 29
+compileSdkVersion: 30
 minSdkVersion: 21
-targetSdkVersion: 29
+targetSdkVersion: 30
```

Comment:
We have to wait for all this related [issues](https://github.com/mozilla-mobile/android-components/issues/8519) to be addressed before we can target Android 11 :)